### PR TITLE
Update input.cc

### PR DESCRIPTION
**Pull request by Aayush-Bachwana-25**
Indented input.cc

### CODE REVIEW

File: pyxel/core/src/pyxelcore/input.cc [hunk 1]
```diff
@@ -7,7 +7,8 @@
 
 namespace pyxelcore {
 
-Input::Input() {
+Input::Input() 
+{
```
Comment (MichaelCurrin) [inline pyxel/core/src/pyxelcore/input.cc:11]: It is not reasonable to change the indentation style to start `{` on a newline by itself.

It adds extra lines without an increase in readability (I think), it is inconsistent with the rest of the project and is an uncommon style (I _very_ rarely see it across any language PHP, C, JS, Go, Rust).

See also Google's C++ style guide https://google.github.io/styleguide/cppguide.html#Namespaces

```cc
// In the .cc file
namespace mynamespace {

void MyClass::Foo() {
}
}  
```

File: pyxel/core/src/pyxelcore/input.cc [hunk 2]
```diff
@@ -18,134 +19,137 @@ Input::Input() {
...
   frame_count_ = frame_count + 1;  // change frame_count to start from 1
 
   SDL_GetGlobalMouseState(&mouse_x_, &mouse_y_);
 
-  mouse_x_ = (mouse_x_ - (window->WindowX() + window->ScreenX())) /
-             window->ScreenScale();
-  mouse_y_ = (mouse_y_ - (window->WindowY() + window->ScreenY())) /
-             window->ScreenScale();
+  mouse_x_ = (mouse_x_ - (window->WindowX() + window->ScreenX())) / window->ScreenScale();
```
Comment (MichaelCurrin) [inline pyxel/core/src/pyxelcore/input.cc:49]: I think the formula was deliberately setup as two lines so make it clear what is on the top and what is on the bottom. Now this is less clear.

File: pyxel/core/src/pyxelcore/input.cc [hunk 3]
```diff
@@ -18,134 +19,137 @@ Input::Input() {
...
-  if (is_mouse_visible_) {
+  if (is_mouse_visible_) 
+  {
     SDL_ShowCursor(true);
-    SDL_SetCursor(mouse_x_ >= 0 && mouse_x_ < window->ScreenWidth() &&
-                          mouse_y_ >= 0 && mouse_y_ < window->ScreenHeight()
-                      ? blank_cursor_
-                      : normal_cursor_);
-  } else {
+    SDL_SetCursor(mouse_x_ >= 0 && mouse_x_ < window->ScreenWidth() && mouse_y_ >= 0 && mouse_y_ < window->ScreenHeight()  ?  blank_cursor_ : normal_cursor_);
```
Comment (MichaelCurrin) [inline pyxel/core/src/pyxelcore/input.cc:56]: This is way too long. On a typical laptop screen mine, this cuts off after `?`.

It's also a lot of more mental effort to read a super long line that one that is broken up into small pieces on 4 lines.

**Review by MichaelCurrin — CHANGES_REQUESTED** [review]
Most of the changes in this PR are not appropriate as they change the style too aggressively, at the cost of readability.

**Comment by kitao** [comment]
I am mechanically adapting clang-format to the code. That is Pyxel's C ++ format policy.

**Pull request closed without merging**